Dockerfile

```diff
@@ -19,33 +19,78 @@ LABEL conda.version=$CONDA_VERSION
 
 # System dependencies
 RUN apt-get -y update
-RUN apt-get -y install vim software-properties-common
+RUN apt-get -y install \
+      software-properties-common \
+      vim
 
 # Install requirements
 RUN apt-get -y install \
-    bzr subversion git-core mercurial libpq-dev libxml2-dev libxslt-dev \
-    libxslt1-dev build-essential postgresql-client libmysqlclient-dev curl \
-    doxygen g++ graphviz-dev libfreetype6 libbz2-dev libcairo2-dev \
-    libenchant1c2a libevent-dev libffi-dev libfreetype6-dev \
-    libgraphviz-dev libjpeg-dev libjpeg8-dev liblcms2-dev libreadline-dev \
-    libsqlite3-dev libtiff5-dev libwebp-dev pandoc pkg-config zlib1g-dev
+      build-essential \
+      bzr \
+      curl \
+      doxygen \
+      g++ \
+      git-core \
+      graphviz-dev \
+      libbz2-dev \
+      libcairo2-dev \
+      libenchant1c2a \
+      libevent-dev \
+      libffi-dev \
+      libfreetype6 \
+      libfreetype6-dev \
+      libgraphviz-dev \
+      libjpeg8-dev \
+      libjpeg-dev \
+      liblcms2-dev \
+      libmysqlclient-dev \
+      libpq-dev \
+      libreadline-dev \
+      libsqlite3-dev \
+      libtiff5-dev \
+      libwebp-dev \
+      libxml2-dev \
+      libxslt1-dev \
+      libxslt-dev \
+      mercurial \
+      pandoc \
+      pkg-config \
+      postgresql-client \
+      subversion \
+      zlib1g-dev
 
 # pyenv extra requirements
 # https://github.com/pyenv/pyenv/wiki/Common-build-problems
 RUN apt-get install -y \
-    make libssl-dev wget llvm libncurses5-dev libncursesw5-dev xz-utils \
-    tk-dev liblzma-dev python-openssl
+      make \
+      libssl-dev \
+      wget \
+      llvm \
+      libncurses5-dev \
+      libncursesw5-dev \
+      xz-utils \
+      tk-dev \
+      liblzma-dev \
+      python-openssl
```

Comment:
I  noticed you sorted the previous `apt-get install`, maybe it's good to sort this one also.